apps/fluent-tester/src/TestComponents/Divider/DividerTest.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { DIVIDER_TESTPAGE } from './consts';
+import { DividerDefault } from './DividerDefault';
+import { Test, TestSection, PlatformStatus } from '../Test';
+
+const dividerSections: TestSection[] = [
+  {
+    name: 'Divider Page',
+    testID: DIVIDER_TESTPAGE,
+    component: DividerDefault,
+  },
+];
+
+export const DividerTest: React.FunctionComponent = () => {
+  const status: PlatformStatus = {
+    win32Status: 'Beta',
+    uwpStatus: 'Experimental',
+    iosStatus: 'Experimental',
+    macosStatus: 'Beta',
+    androidStatus: 'Backlog',
+  };
+
+  const description = 'component-description';
```

Comment:
updating the component description here. Generally we add the same thing as in spec prepare.